modules/rosa-hcp-upgrading-cli-tutorial.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * upgrading/rosa-hcp-upgrading.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-upgrading-cli_{context}"]
+= Upgrading with the ROSA CLI
+
+You can manually upgrade a {hcp-title} cluster by using the ROSA CLI. This method schedules the cluster for an immediate upgrade if a more recent version is available.
+
+.Prerequisites
+
+* You have installed and configured the latest version of the ROSA CLI.
+
+.Procedure
+
+. Verify the current version of your cluster by running the following command:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name|cluster_id> <1>
+----
+<1> Replace `<cluster_name|cluster_id>` with the cluster name or the cluster ID.
+
+. Verify that a control plane upgrade is available by running the following command:
++
+[source,terminal]
+----
+$ rosa list upgrade --cluster=<cluster_name|cluster_id>
+----
++
+The command returns a list of upgradeable versions, including the recommended version.
++
+.Example output
++
+[source,terminal]
+----
+VERSION  NOTES
+4.14.8   recommended
+4.14.7
+4.14.6
+----
+
+. Verify that a machine pool upgrade is available by running the following command:
++
+[source,terminal]
+----
+$ rosa list upgrade --cluster <cluster-name> --machinepool <machinepool_name>
+----
++
+The command returns a list of upgradeable versions, including the recommended version.
++
+.Example output
++
+[source,terminal]
+----
+VERSION  NOTES
+4.14.5   recommended
+4.14.4
+4.14.3
+4.14.2
+4.14.1
+----
++
+[NOTE]
+====
+Your machine pools will not show upgrade versions beyond your cluster's current version. Ensure your cluster is up to date first.
+====
+
+. Upgrade your cluster with one of the following options:
+** Upgrade the cluster's hosted control plane by running the following command:
++
+[source,terminal]
+----
+$ rosa upgrade cluster -c <cluster_name> --control-plane [--schedule-date=XX --schedule-time=XX] [--version 4.14.8]
+----
++
+Your hosted control plane is now scheduled for an upgrade.
+
+** Upgrade a specific machine pool on your cluster by running the following command:
++
+[source,terminal]
+----
+$ rosa update machinepool -c <cluster_name> <your_machine_pool_id> [--version 4.14.8]
+----
++
+Your machine pool is now scheduled for an upgrade.
+//
+// ** Upgrade both the hosted control plane and all attached machine pools by running the following command:
+// +
+// [source,terminal]
+// ----
+// $ rosa upgrade cluster -c <cluster_name> [--schedule-date=XX --schedule-time=XX] [--version 4.14.8]
```

Comment:
as mentioned, this doesn't exist at the moment, I assume the // means commented out, so not appearing in the published doc right?

Reply:
Yup, I can totally remove it if we want to publish the rest of the updates until this is ready, but right now, it's totally hidden.